core-java-modules/core-java-uuid/src/main/java/com/baeldung/timebaseduuid/UUIDCreator.java

```diff
@@ -0,0 +1,41 @@
+package com.baeldung.timebaseduuid;
+
+import com.github.f4b6a3.uuid.UuidCreator;
+
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class UUIDCreator {
+
+    public static void main(String[] args) throws InterruptedException {
+
+        int threadCount = 128;
+        int iterationCount = 100_000;
+        ConcurrentMap<UUID, Long> uuidMap = new ConcurrentHashMap<>();
+        AtomicLong collisionCount = new AtomicLong();
+        long startNanos = System.nanoTime();
+        CountDownLatch endLatch = new CountDownLatch(threadCount);
+
+        for (long i = 0; i < threadCount; i++) {
+            final long threadId = i;
```

Comment:
I'm not sure this needs to be `final`. In Java 8 it's _effectively final_ which I think means we can omit the `final` keyword here.